bloom/commands/git/import_upstream.py

```diff
@@ -38,6 +38,7 @@
 import tarfile
 import tempfile
 
+from io import open
```

Comment:
hmmm yeah that's unfortunate, Python2 and encoding is always a pain :(
Well I guess, my recommendation would then be: if its confirmed that we need this import here let's call out `io.open` explicitely otherwise let's ditch the import in this file.
And for the other files lets not override the default to avoid unwanted side effects (having to convert every string to unicode) and just use io.open where it's necessary.

Thanks for the detailed explanation!